src/Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: lint
+
```

Comment:
nit: extra new line

Reply:
I'll readily admit I have extremely little experience with `Makefile` formatting best practices. I thought this empty line here is a Good Thing (TM). Point me to some good guide? Quick Google search didn't give me anything.

Reply:
You're right, looking at ftp://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_4.html#SEC33 there doesn't seem to be a particular style. In that page examples they appear before, after, all targets on one line, with and without blank line, etc.. :(

I don't really care, I just like to keep them close to the target to avoid them getting deleted by mistake. But one single line is fine, I'd be more annoyed with multiple PHONY on one line.